GoogleForCreators/web-stories-wp
# Workspace: Search functionality inside new Font Picker

## Summary

Adds search functionality to new font picker.

## Relevant Technical Choices

Filters the matched fonts based on what is included in the search string. Builds on top of the code from https://github.com/google/web-stories-wp/pull/2249. The delta in this PR will look bigger until that is approved.

## User-facing changes

**None at the moment because it's all behind a feature flag**

But when enabled:

Adds a search input that stays fixed to the font drop down panel that lets users search for fonts.

<img width="364" alt="Screen Shot 2020-06-08 at 2 09 20 PM" src="https://user-images.githubusercontent.com/1738349/84083076-8d639380-a9a6-11ea-8007-87503d1ea3cd.png">


## Testing Instructions

1. Enable the new font picker
2. Select a text box and choose the font picker from the panel
3. Type the name of a font partially and then see the list filter 

---



#30
